### ISSUE TITLE
FF default bookmarks [solved: not a privacy issue: just delete them or look at Enterprise Policy]

### ISSUE THREAD

**Issue by m77e4t** [arkenfox/user.js#1318]
As said in title. The issue might be minor but is somewhat resonable. 
The listed bookmarks are not that useful and I had to manually delete them. If the bookmarks are deliberately kept, then I have no problem. If majority of users want them by default on arkenfox, then keep it.

![Screenshot from 2021-12-31 12-51-05](https://user-images.githubusercontent.com/96620334/147865669-f4acf358-0cd7-4b57-92f2-4034e806f272.png)
![Screenshot from 2022-01-02 04-43-25](https://user-images.githubusercontent.com/96620334/147865676-402dce34-b14e-44d4-b79d-39cfb2a0c122.png)

**Comment by Thorin-Oakenpants** [comment]
IDC what default tiles are in activity stream and I am not interested in following the changes in order to maintain it. What you get in a new profile are some defaults "top sites"

e.g. `browser.newtabpage.activity-stream.default.sites`
```
https://www.youtube.com/,
https://www.facebook.com/,
https://www.amazon.com/,
https://www.reddit.com/,
https://www.wikipedia.org/,
https://twitter.com/
```

We already blank this, but that pref is not (or no longer) applicable to what is on AS (in fact I am struggling now to know what this does, it doesn't seem to stop urlbar suggestions)

We used to have some AS prefs where we used `""` or "{}" etc but it all got messy
- https://github.com/arkenfox/user.js/issues/1072#issuecomment-744028751
- https://github.com/arkenfox/user.js/commit/9d74cb95266f60e73d4aba4a73e0623e4721082c
- not going to dig any deeper

tl;dr: they are not a privacy issue, they are simply some defaults. Who am I to decide that a reddit or amazon or wikipedia tile should be removed. AF does not enable AS by default, and for those that do, they can easily remove shit from THEIR tiles as THEY see fit

In removing items, it writes to `browser.newtabpage.blocked`, e.g.
```
"{\"26UbzFJ7qT9/4DhodHKA1Q==\":1,\"4gPpjkxgZzXPVtuEoAL9Ig==\":1,\"eV8/WsSLxHadrTL1gAxhug==\":1,\"gLv0ja2RYVgxKdp0I5qwvA==\":1,\"K00ILysCaEq8+bEqV/3nuw==\":1,\"T9nJot5PurhJSy8n038xGA==\":1}");
```

I'm not doing that

Maybe @Jee-Hex has something for you, but for AF this is out of scope

**Comment by Thorin-Oakenpants** [comment]
The row that contains these is "shortcuts"
= `browser.newtabpage.activity-stream.feeds.topsites`

![shortcuts](https://user-images.githubusercontent.com/16656956/147866604-88a60686-3ed4-4e34-8459-90b9eed77940.png)

https://github.com/arkenfox/user.js/blob/7016c2050d20d53242c30eaf0b3beca947e172f6/user.js#L98-L100

Note:
- disable _**SOME**_ Activity Stream items 
- [SETTING] Home>Firefox Home Content>...  to **_show/hide what YOU want_**
- `browser.newtabpage.activity-stream.feeds.topsites` is not included

**Comment by m77e4t** [comment]
We had a misunderstanding here, I wasn't talking about tiles, but specifically bookmarks from bookmarks manager. It may be due to I worded it incorrectly and edit the images properly.

![Screenshot from 2022-01-02 11-49-44](https://user-images.githubusercontent.com/96620334/147867945-73b8bec7-b53a-4f73-bc68-4ec1b62b10c7.png)
 
The first image is from normal default firefox from mozilla. In it, default bookmarks are already installed by mozilla. They can be seen in the upper right-hand corner. 

![Screenshot from 2022-01-02 11-45-22](https://user-images.githubusercontent.com/96620334/147867997-33939dbf-2911-447e-a756-95c2ab8041b5.png)

Second Image is from arkenfox.js. Pocket, tiles like facebook, amazon, wikipedia etc and the search widget in middle are removed by default arkenfox.js. Though, in the upper right-hand corner, bookmarks from bookmarks manager can be seen. They are not specifically a privacy issue, but there is no use of them, especially something like 'getting started with firefox' bookmark. While, manually deleting them, I accidentally clicked (butterfingered) on one of them and was redirected to the official mozilla website. As mozilla website is not that bad from a privacy standpoint, it is not a big deal. As it happened with, it may have happened with others too and might happen again. It was my mistake of accidentally clicking it, but they may be removed as they pretty much of no use.

**Comment by Thorin-Oakenpants** [comment]
OK, my bad. Still not a privacy issue, just delete them. Am knee deep in something else right now, so will come back to this another time

> Pocket, tiles like facebook, amazon, wikipedia etc and the search widget in middle are removed by default arkenfox.js.

I'm not sure they are, it's only because we change
- user_pref("browser.startup.page", 0);
- user_pref("browser.startup.homepage", "about:blank");

If you change `Home>New Windows and Tabs>Homepage and new windows`  to `Firefox (default)` (and shortcuts are enabled, which we don't have in the user.js), they are there - as per my image two posts up which was done in my arkenfoxed FF

**Comment by Jee-Hex** [comment]
The [`NoDefaultBookmarks`](https://github.com/mozilla/policy-templates#nodefaultbookmarks) policy may be useful if you don't want to do this manually for every new profile.

**Comment by m77e4t** [comment]
Basic question, will using it make me more unique than other arkenfox.js users.

Even if it did not, I think I will go with the current method of manually deleting them.

**Comment by Thorin-Oakenpants** [comment]
> will using it make me more unique

no, bookmarks are not enumerable

**Comment by m77e4t** [comment]
K. If you want, you can close the issue.
